myModule.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
i do not know. i would guess this file was automatically generated when i ran cmd check. should i remove this file?